.babelrc

```diff
@@ -2,21 +2,34 @@
   "presets": [
     "@babel/preset-react",
     "@babel/preset-flow",
-    [ "@babel/preset-env", {
-      "modules": false,
-    }]
+    [
+      "@babel/preset-env",
+      {
+        "modules": false,
+      }
+    ]
   ],
   "plugins": [
     "@babel/plugin-transform-exponentiation-operator",
     "@babel/plugin-proposal-class-properties",
     "@babel/plugin-syntax-dynamic-import",
-    ["@babel/plugin-proposal-object-rest-spread", { "useBuiltIns": true }],
+    [
+      "@babel/plugin-proposal-object-rest-spread",
+      {
+        "useBuiltIns": true
+      }
+    ],
     "babel-plugin-add-react-displayname"
   ],
+  "sourceType": "unambiguous",
```

Comment:
This is the 1 actual change here. VS code formatted the rest like this for me and it looks better so i didn't undo it